ui/apps/ui/src/app/collections/services/redirect.service.ts

```diff
@@ -20,7 +20,7 @@ export class RedirectService {
     }
 
     const encodedPv = encodeURIComponent(
-      'search/' + this._customRoute.collection()
+      '/search/' + this._customRoute.collection()
```

Comment:
Forward slash here should not be added. We already informed our partners that this interface would not change (also it would require changes in MP)

Reply:
done